go-mode.el

```diff
@@ -1835,7 +1835,7 @@ If ARG is non-nil, anonymous functions are skipped."
 If ARG is non-nil, anonymous functions are skipped."
   (interactive "P")
   (go-goto-function-name arg)
-  (forward-word 1)
+  (forward-word-strictly 1)
```

Comment:
Well, the Emacs folks clearly have gone insane. I will fix this myself, in a backwards compatible way. *Someone* has to be backwards compatible, right? :-)